modules/mukurtu_migrate/src/Plugin/migrate/source/DictionaryWord.php

```diff
@@ -28,23 +28,12 @@
  *   id = "dictionary_word"
  * )
  */
+class DictionaryWord extends Node {
 
-class DictionaryWord extends Node
-{
   /**
    * {@inheritdoc}
    */
-  public function query() {
-    $query = parent::query();
-    $query->condition('n.type', 'dictionary_word');
-    return $query;
-  }
```

Comment:
The parent class does this via a `node_type` configuration. See `migrate_plus.migration.mukurtu_cms_v3_dictionary_word.yml` where I set `node_type: dictionary_word`.